code/datums/mutations/sight.dm

```diff
@@ -95,6 +95,13 @@
 
 	user_mob.adjustOrganLoss(ORGAN_SLOT_EYES, eye_damage)
 
+/datum/mutation/human/thermal/on_losing(mob/living/carbon/human/owner, mob/user = usr)
+	if(..())
+		return
+	REMOVE_TRAIT(owner, TRAIT_THERMAL_VISION, GENETIC_MUTATION)
+	owner.update_sight()
+	to_chat(user, span_notice("You can no longer see the heat rising off of your skin..."))
```

Comment:
remove this and move to `text_lose_indication`

https://github.com/tgstation/tgstation/blob/55b92786dcf0f9c5172efd111a6ecbf83ec0dae2/code/datums/mutations/_mutations.dm#L13

Reply:
done, didnt even know that var existed since nothing else on that page used it